app/src/main/java/org/mozilla/fenix/HomeActivity.kt

```diff
@@ -20,6 +20,7 @@ import androidx.navigation.NavDirections
 import androidx.navigation.fragment.NavHostFragment
```

Comment:
Did you file a bug for the Fenix team for the behavior where the BookmarksFragment shows "Firefox Preview" briefly before the proper title?

Reply:
https://github.com/mozilla-mobile/fenix/issues/7789